core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -180,6 +180,20 @@ class JdbcAdapterTest {
             + "WHERE \"product_id\" = 1) AS \"t3\"");
   }
 
+  @Test void testUnionIncludingCte() {
+    CalciteAssert.model(FoodmartSchema.FOODMART_MODEL)
```

Comment:
This test case can be executed without any modification. Please reconfirm it.

Reply:
I also tested it a few minutes ago. I can confirm that this PR is no longer required.